src/knot/events/replan.c

```diff
@@ -83,7 +83,7 @@ static bool can_expire(const zone_t *zone)
 }
 
 /*!
- * \brief Replan events that depend on zone timers (REFRESH, EXPIRE, FLUSH, RESALT, PARENT DS QUERY).
+ * \brief Replan events that depend on zone timers (REFRESH, EXPIRE, FLUSH, RE_SALT, PARENT DS QUERY).
```

Comment:
This would probably be better as `RE-SALT`, but I'm not sure what it is...

Reply:
Please, keep RESALT. It's our "idiom" :-)